src/sentry/logging/handlers.py

```diff
@@ -48,13 +48,23 @@ def __call__(self, logger, name, event_dict):
 
 class StructLogHandler(logging.StreamHandler):
     def emit(self, record, logger=get_logger()):
+        # This may seem a little gross, but when logging with the RootLogger,
+        # anything passed into 'extra' gets turned into a real kwarg
+        throwaways = [
```

Comment:
Yeah, sorry, I mean, a comment linking to where these come from would be useful. I assume you got the list somewhere from here: https://github.com/python/cpython/blob/2.7/Lib/logging/__init__.py